File: ObjectiveC/JWSLandscapeObjectiveC/JWSLandscapeObjectiveC/ViewController.m
//
//  ViewController.m
//  JWSLandscapeObjectiveC
//
//  Created by Clint on 2018. 1. 10..
//  Copyright © 2018년 clintjang. All rights reserved.
//

#import "ViewController.h"
#import "LandscapeController.h"
#import "DataManager.h"

@interface ViewController ()
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)showLandscapeController:(id)sender {
    [DataManager sharedManager].isEnableLandscape = YES;
    [DataManager sharedManager].forcedLandscape = YES;  
    LandscapeController *controler = [self.storyboard instantiateViewControllerWithIdentifier:@"LandscapeController"];
    [self presentViewController:controler animated:true completion:^{
        
    }];
}

@end

File: JWSLandscape/ViewController.swift
//
//  ViewController.swift
//  JWSLandscape
//
//  Created by Clint on 2017. 10. 17..
//  Copyright © 2017년 Clint. All rights reserved.
//

import UIKit

class ViewController: BaseViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func showLandscapeController(_ sender: Any) {
        DataManger.shared.isEnableLandscape = true
        let controller:UIViewController = (self.storyboard?.instantiateViewController(withIdentifier: "LandscapeController"))!
        self.present(controller, animated: true) {
            
        }
    }
    
}

